script/cadence/contracts/FlowColdStorageProxy.cdc

```diff
@@ -139,14 +139,15 @@ pub contract FlowColdStorageProxy {
     //
     // And, finally, the FlowColdStorageProxy.Vault itself is made directly
     // accessible via /public/flowColdStorageProxyVault.
-    pub fun setup(payer: AuthAccount, publicKey: [UInt8]) {
+    pub fun setup(payer: AuthAccount, publicKey: [UInt8]): Address {
         let acct = AuthAccount(payer: payer)
         acct.save(<- create Vault(publicKey: publicKey), to: self.VaultCapabilityStoragePath)
         acct.unlink(/public/flowTokenReceiver)
         acct.link<&{FungibleToken.Receiver}>(/public/defaultFlowTokenReceiver, target: /storage/flowTokenVault)!
 		acct.link<&{FungibleToken.Receiver}>(/public/flowTokenReceiver, target: self.VaultCapabilityStoragePath)!
 		acct.link<&Vault>(self.VaultCapabilityPublicPath, target: self.VaultCapabilityStoragePath)!
         emit Created(account: acct.address, publicKey: String.encodeHex(publicKey))
+        return acct.address
```

Comment:
do we need to change service code due to this newly return?

Reply:
Since it is only returning the address, not a resource then the caller can ignore this.